src/middleware/packages/activitypub/services/activitypub/subservices/inbox.js

```diff
@@ -45,6 +45,13 @@ const InboxService = {
         throw new E.UnAuthorizedError('INVALID_ACTOR', 'Activity actor is not the same as the posting actor');
       }
 
+      // confirm that the account is not a tombstone
+      const account = await ctx.call('auth.account.findByUsername', { username: activity.username });
+      if (account && account.deletedAt) {
+        const error = new Error(`The account has been deleted`);
+        error.code = 410;
```

Comment:
Please use the MoleculerError to include directly the HTTP code:
https://github.com/moleculerjs/moleculer/blob/4014a8c873956346c17357701433f0cc9c01be8e/src/errors.js#L53
For 404 error, there is a dedicated moleculer-web error:
https://github.com/moleculerjs/moleculer-web/blob/c6ec80056a64ea15c57d6e2b946ce978d673ae92/src/errors.js#L127

Reply:
fixed.